src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -139,6 +142,34 @@ public String getDataInventory(String params) {
         return data;
     }
 
+    public String getReloadToken(InputStream input) {
+        try {
+            BufferedReader in = new BufferedReader(new InputStreamReader(input));
+            String currentLine;
+            while ((currentLine = in.readLine()) != null) {
+                if (currentLine.contains("&reloadToken=")) {
+                    String token;
+                    int start;
+                    int end;
+                    start = currentLine.indexOf("reloadToken=")+12;
+                    end = currentLine.indexOf('"',start);
+                    if (start != 0 && end !=0) {
```

Comment:
Shouldn't this check be against -1? string.indexOf returns -1 if the sequence isn't found, not 0